srcpkgs/libclang-python3/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'libclang-python3'
+pkgname=libclang-python3
+version=3.9.0
+revision=1
+wrksrc="libclang-py3-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-setuptools"
+short_desc="Python3 bindings for libclang"
+maintainer="shizonic <realtiaz@gmail.com>"
+homepage="https://github.com/Sublimelibclang-py3/libclang-py3"
+license="Unknown"
```

Comment:
setup.py says BSD license (which doesn't exist, there lots of BSD-style licenses) but the files have `NCSA` header

Reply:
So that means? Which license should I use?

Reply:
I can't find any license file in the repo. So do I have to create it by myself?

Reply:
Hm, the sources files say `This file is distributed under the University of Illinois Open Source License.`, sooooo

Reply:
Okay. Will do that. Thanks!